PyVideoMEG/bin/pvm_export.py

```diff
@@ -30,8 +30,11 @@
 import shutil
 import numpy
 
-import Image, ImageDraw, ImageFont
-import StringIO
+from PIL import Image, ImageDraw, ImageFont
+try:
+    from StringIO import StringIO
```

Comment:
Might as well do `from BytesIO import BytesIO` on Python 2, as well -- then you don't need to triage later

Reply:
Great point and works.
Changed!